hs_file_types/tests/test_view_functions.py

```diff
@@ -292,15 +292,19 @@ def test_create_fileset_aggregation(self):
     def test_remove_aggregation(self):
         # here we are testing the remove_aggregation view function
 
-        self.create_composite_resource(file_to_upload=self.netcdf_file)
```

Comment:
did the code around file_to_upload get removed as well?  I'm not seeing the deletion in here.  It could be removed from the create_resource method in hs_core/hydroshare/resource.py.  I guess that would set off a chain reaction to remove those tests for the other types, but it shouldn't be too bad to update those tests.  It would be a valuable cleanup.  Let me know if you'd like to take it on here or if we should split into another issue as it's not directly related to your work here.

Reply:
ahh, I knew I had made a comment about this in a review.  I've talked myself out of this.